### PR TITLE
[DEVELOP-1138] Small revisions to component UX

### PR DESCRIPTION
**Pull request by trydionel**
* Change default aspect ratio from 16:9 to 4:3. Gives more room for embedded content footers.
* Improve empty state for EmbeddedContent component when a blank/null URL is provided.
* Make "Remove" menu action scarier

### CODE REVIEW

**Comment by trydionel** [comment]
@percyhanna Last one, I promise 😇

File: src/components/EmbeddedContent.tsx [hunk 1]
```diff
@@ -1,5 +1,26 @@
 import React from "react";
 
+interface AspectRatioProps {
+  aspectRatio: number;
+  children: any;
+}
+
+const AspectRatio = ({ aspectRatio, children }: AspectRatioProps) => (
```
Comment (percyhanna) [inline src/components/EmbeddedContent.tsx]: Can we make this its own export in its own file?

Comment (trydionel) [reply src/components/EmbeddedContent.tsx]: Sure thing. I'll update after some calls this morning.

Comment (trydionel) [reply src/components/EmbeddedContent.tsx]: Revised

**Review by percyhanna — APPROVED** [review]

**Review by cornu-ammonis — APPROVED** [review]
I reviewed this as part of the reviews for the extensions in which it is used. It looks good from a security perspective

**Pull request merged**